### PR TITLE
Create Log section, Closes #28

### LINKED ISSUE #28: Log section
comment from @diego-G:
> After LiskHQ/lisk: Issue #2148

comment from @shuse2:
> This should be carefully considered, because once documented as interface, it's really hard to change

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #28

### CODE REVIEW

File: configuration.md [hunk 1]
```diff
@@ -247,3 +248,90 @@ Next snippet highlights the essential parameters to enable SSL security on your
...
+
+These log messages are grouped in different log levels, which makes it easy to define the level of detail for the logs.
+
+We use [Bunyan](https://github.com/trentm/node-bunyan) as logging library. Bunyan allows simple and fast JSON logging for Node.js services.
+
+### Log Levels
+
+| Log Level | Description                                                                                                                 |
+| ----------| ----------------------------------------------------------------------------------------------------------------------------|
+| None(>60) | No events are logged.                                                                                                       |
```
Comment (shuse2) [inline configuration.md]: None doesnt have log level number

File: setup/source.md [hunk 2]
```diff
@@ -262,7 +261,6 @@ Before proceeding, determine whether you wish to connect your node to the Mainne
 
 ```bash
 git clone https://github.com/LiskHQ/lisk-sdk.git
```
Comment (shuse2) [inline setup/source.md]: this should be `lisk-core`?

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: setup/source.md (hunk 2 above)
Comment (Tschakki) [reply setup/source.md]: this is updated in #217

File: monitoring.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+# Log Monitoring
+
+The [log files](configuration.md#file-log-stream) provide a chronological list of all events that happened on a node.
```
Comment (nazarhussain) [inline monitoring.md]: > list of all events 

Here events are something very specific to NodeJS events or module events we specified in our protocol. We rather use different term here. 

```suggestion
The [log files](configuration.md#file-log-stream) provide a chronological list of activities that happened on a node.
```

File: monitoring.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
+These events can be analyzed further in order to provide monitoring for a particular node or for the network in general.
```
Comment (nazarhussain) [inline monitoring.md]:
```suggestion
These activities can be analyzed further in order to provide monitoring for a particular node or for the network in general.
```

File: configuration.md [hunk 5]
```diff
@@ -247,3 +248,90 @@ Next snippet highlights the essential parameters to enable SSL security on your
 ```
 
 To verify all you have properly configured your node, open the web client using `https://MY_IP_OR_HOST`. You should now see a secure SSL connection.
+
+## Logging 
+
+For monitoring or debugging your node, Lisk Core tracks all important events that happen on the node by creating log messages for them.
```
Comment (nazarhussain) [inline configuration.md]:
```suggestion
For monitoring or debugging your node, Lisk Core tracks all activity that happen in the node by creating log messages for them.
```

**Review by nazarhussain — CHANGES_REQUESTED** [review]

**Review by nazarhussain — DISMISSED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**